src/mumble/AudioOutputToken.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+#define MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+
+#include <AudioOutput.h>
+#include <AudioOutputToken.h>
+
+class AudioOutputBuffer;
+
+class AudioOutputToken {
+private:
+	AudioOutputBuffer *m_buffer;
+
+	friend class AudioOutput;
+
+public:
+	AudioOutputToken(AudioOutputBuffer *buffer) : m_buffer(buffer){};
```

Comment:
```suggestion
	AudioOutputToken(AudioOutputBuffer *buffer = nullptr) : m_buffer(buffer){};
```
Make it default-constructible

Reply:
I have instead created the default constructor ``AudioOutputToken()`` which also means that members of the type AudioOutputToken are implicitly initialized without adding them to the initializer list

Reply:
If I understood you correctly, the result of both approaches should be exactly the same :)